<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ultimate Blindtest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="global_VLayout">
      <item>
       <layout class="QHBoxLayout" name="menuViewer_HLayout">
        <item>
         <widget class="QFrame" name="menuFrame">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="categoryLabel">
             <property name="text">
              <string>Select a category :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="categoryCombo"/>
           </item>
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>115</red>
                   <green>210</green>
                   <blue>22</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>115</red>
                   <green>210</green>
                   <blue>22</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>115</red>
                   <green>210</green>
                   <blue>22</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start Game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>239</red>
                   <green>41</green>
                   <blue>41</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>239</red>
                   <green>41</green>
                   <blue>41</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>239</red>
                   <green>41</green>
                   <blue>41</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="text">
              <string>Stop Game</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="verticalWidgetAnswers" native="true">
             <layout class="QVBoxLayout" name="verticalLayoutAnswers">
              <item>
               <widget class="QLineEdit" name="answerArtist">
                <property name="placeholderText">
                 <string>Artist</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="answerSong">
                <property name="placeholderText">
                 <string>Song name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="answerIn">
                <property name="placeholderText">
                 <string>Heard in</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="answerLink">
                <property name="placeholderText">
                 <string>Youtube link</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="answerTime">
                <property name="placeholderText">
                 <string>Guess time</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editFieldsButton">
                <property name="text">
                 <string>Edit song</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="playersList"/>
           </item>
           <item>
            <widget class="QSlider" name="difficultySlider">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="teamName">
             <property name="placeholderText">
              <string>Team name</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="3" column="0">
              <widget class="QPushButton" name="resetScoreButton">
               <property name="text">
                <string>Reset Scores</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="resetDoneButton">
               <property name="text">
                <string>Reset Done</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="addTeamButton">
               <property name="text">
                <string>Add Team</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="removeTeamButton">
               <property name="text">
                <string>Remove Team</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="revealButton">
               <property name="text">
                <string>Reveal Song</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="addTimeButton">
               <property name="text">
                <string>Add 15sec</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="viewerFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cntdownScore_HLayout">
        <item>
         <widget class="QLabel" name="countdownLabel">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>150</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>30</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="teamTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="manualCinemaButton">
            <property name="text">
             <string>Cinema Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clickAdButton">
            <property name="text">
             <string>Click Ad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plusOneButton">
            <property name="text">
             <string>+1 Point</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="minusOneButton">
            <property name="text">
             <string>-1 Point</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zeroPointButton">
            <property name="text">
             <string>Back to 0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
